src/methods/function.js

```diff
@@ -506,9 +506,13 @@ export function SetFunctionName(
     // b. If description is undefined, let name be the empty String.
     if (description === undefined) {
       name = realm.intrinsics.emptyString;
+    } else if (name instanceof AbstractValue) {
+      name = realm.createAbstract(new TypesDomain(StringValue), ValuesDomain.topVal, [name], ([lnode, rnode]) =>
```

Comment:
This looks wrong. `[name]` has a different number of elements than `[lnode, rnode]`. What is `+` applied to?